### PR TITLE
Fix @autoserialize problem with unknown module

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: szalpal <mszolucha@nvidia.com>

## Category:

**Bug fix** (*non-breaking change which fixes an issue*)

## Description:
When `autograph` has been introduced to DALI, we've faced the problem with autoserialization. Turns out, that `autograph` in runtime swaps the modules using `six` - probably to maintain compatibility between Python2 and Python3. Therefore, when traversing the module tree in search for `@autoserialize` decorator, inspect has not been able to find some modules and was prompting an error.

The solution is that if given module is unavailable, it is not traversed any more. DALI won't be able to retrieve `@autoserialize` from there anyhow, so no harm done.

### Tests:

- [x] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [x] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by szalpal** [comment]
!build

**Review by JanuszL — APPROVED** [review]

File: dali/python/nvidia/dali/_utils/autoserialize.py [hunk 1]
```diff
@@ -30,7 +30,12 @@ def _discover_autoserialize(module, visited):
     """
     assert module is not None
     ret = []
-    module_members = inspect.getmembers(module)
+    try:
```
Comment (JanuszL) [inline dali/python/nvidia/dali/_utils/autoserialize.py:33]: Can you add a simple test with `import xyz` inside?

**Comment by szalpal** [comment]
!build

File: dali/python/nvidia/dali/_utils/autoserialize.py (hunk 1 above)
Comment (szalpal) [reply dali/python/nvidia/dali/_utils/autoserialize.py:33]: Done

File: dali/test/python/autoserialize_test/custom_module_inside.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from nvidia.dali.plugin.triton import autoserialize
+from nvidia.dali import pipeline_def
+
+
+@autoserialize
+@pipeline_def(batch_size=1, num_threads=1, device_id=0)
+def func_under_test():
+    import numpy as np  # noqa: F401
```
Comment (JanuszL) [inline dali/test/python/autoserialize_test/custom_module_inside.py:8]: It is likely that the test env may have numpy installed. How about `import doesnt_exist`?

**Pull request merged**